lib/upgrade/converters/policy_test.go

```diff
@@ -387,3 +388,62 @@ func TestCanConvertV1ToV3Policy(t *testing.T) {
 		})
 	}
 }
+
+var policyTableBackend = []struct {
+	description string
+	v1KVP       *model.KVPair
+	v3API       apiv3.GlobalNetworkPolicy
+}{
+	{
+		description: "policy with non-strictly masked CIDR should get converted to strictly masked CIDR in v3 API",
+		v1KVP: &model.KVPair{
+			Key: model.PolicyKey{
+				Name: "MaKe.-.MaKe",
+			},
+			Value: &model.Policy{
+				Order: &order1,
+				// Source Nets selector in V1ModelInRule1 and V1ModelEgressRule1 are non-strictly masked CIDRs.
+				InboundRules:   []model.Rule{V1ModelInRule1},
+				OutboundRules:  []model.Rule{V1ModelEgressRule1},
+				Selector:       "thing == 'value'",
+				DoNotTrack:     true,
+				PreDNAT:        false,
```

Comment:
I'm confused also about the purpose of this new test.  Does `PreDNAT: false` here mean that this *isn't* testing your fix?

Reply:
I'm not sure what happened here. I think I must have copied from above and paired down stuff but then got distracted with something else and forgot to come back and make this be what I intended.  Wow. really sorry.